test/lsp/lsp_formatting.ml

```diff
@@ -401,10 +401,10 @@ let%expect_test "formatting-request-whole-program" =
   end_with_postproc [%expect.output];
   [%expect {|
     {"params":{"diagnostics":[],"uri":"file://__rootdir__/superbol.toml"},"method":"textDocument/publishDiagnostics","jsonrpc":"2.0"}
-    src/lsp/cobol_ast/raw_misc_sections_visitor.ml:66:
+    raw_misc_sections_visitor.ml:0:
```

Comment:
@lefessan What was the problem of having precise source locations here?
To me the occasional need for an auto-promotion does not warrant the ugly hack used in this PR to remove the info.

Reply:
These locations are very handy when implementing the partial visitors, even and in particular in the tests.